prov/gni/src/gnix_mr_cache.c

```diff
@@ -373,6 +622,15 @@ static inline int __insert_entry_into_stale(
 	RbtStatus rc;
 	int ret = 0;
 
+	if ((entry->state == GNIX_CES_INUSE_UNMAPPED) ||
+	    (entry->state == GNIX_CES_STALE_UNMAPPED)) {
+		GNIX_INFO(FI_LOG_MR, "entry (%p) unmapped, not inserting"
+			  " into stale %llx:%llx", entry,
```

Comment:
Could we simply make UNMAPPED a flag rather than a state? It doesn't feel like a state. This is a nit. It doesn't really affect how we do things here at the moment.